projects/ngx-wig/src/lib/ngx-wig.component.ts

```diff
@@ -167,6 +167,8 @@ export class NgxWigComponent
     } else {
       this.pasteHtmlAtCaret(text);
     }
+
+    this.onContentChange(this.container.innerHTML)
```

Comment:
```suggestion
    this.onContentChange(this.container.innerHTML);
```

Reply:
Oh dear, that's embarrassing.... Thanks @bampakoa!